packages/pn-commons/src/utility/notification.utility.ts

```diff
@@ -386,10 +391,7 @@ export function getLegalFactLabel(
       'detail.timeline.legalfact.analog-failure-delivery',
       'Deposito di avvenuta ricezione'
     );
-  } else if (
-    timelineStep.category === TimelineCategory.ANALOG_FAILURE_WORKFLOW &&
-    (timelineStep.details as AnalogWorkflowDetails).getGeneratedAarUrl
-  ) {
+  } else if (timelineStep.category === TimelineCategory.ANALOG_FAILURE_WORKFLOW) {
```

Comment:
Why do you remove the (timelineStep.details as AnalogWorkflowDetails).getGeneratedAarUrl condition?

The TimelineCategory.ANALOG_FAILURE_WORKFLOW can occurs in several ways. Removing the check on generatedAarUrl you are telling that ALL the ANALOG_FAILURE_WORKFLOW must have the same legal fact label and this is wrong!
Only when we have generatedAarUrl, we have an AAR and we have to set this label, in the other cases the default "Attestazione opponibile a terzi" is good

Reply:
in our case the variable getAarUrl is never evaluated and therefore never enters the if

Reply:
@leleOFA do you have checked the timeline element that corresponds to the ANALOG_FAILURE_WORKFLOW category?

Reply:
corresponds to that, in fact by removing the condition that I eliminated in the commit the right copy is inserted

Reply:
if I insert the condition without the generate but leaving the old one and adding a new one it inserts the copy correctly